docs/en/Clients-Python.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: global
+title: Python Client
+nickname: Python Client
+group: Clients
+priority: 5
+---
+
+Alluxio has a [Python Client](https://github.com/Alluxio/alluxio-py) for interacting with Alluxio through its
+[REST API](Clients-Rest.html). The Python client exposes an API similar to the [native Java API](Clients-Java-Native.html).
+See the [doc](http://alluxio-py.readthedocs.io) for detailed documentation about all available
+methods. See the [example](https://github.com/Alluxio/alluxio-py/blob/master/example.py) of how to perform basic filesystem
+operations in Alluxio.
+
+# Alluxio Proxy dependency
+
+The Python client talks to Alluxio through the REST API provided by the Alluxio proxy.
```

Comment:
NIT: "talks to Alluxio" -> "interacts with Alluxio"

Reply:
done